src/wallet/wallettool.cpp

```diff
@@ -105,7 +105,7 @@ static void WalletShowInfo(CWallet* wallet_instance)
 
 bool ExecuteWalletToolFunc(const ArgsManager& args, const std::string& command, const std::string& name)
 {
-    fs::path path = fs::absolute(name, GetWalletDir());
+    fs::path path = fs::absolute((name == "") ? GetWalletDir() : (GetWalletDir() / name));
```

Comment:
Why not just `GetWalletDir() / name` even if `name` is empty?

Reply:
In commit "refactor: replace boost::filesystem with std::filesystem" (f449c26f30bb8e223d4d7833c579d555a0005daa)

re: https://github.com/bitcoin/bitcoin/pull/20744#discussion_r547333061

> Why not just `GetWalletDir() / name` even if `name` is empty?

See https://github.com/bitcoin/bitcoin/pull/20744#issuecomment-750430904, this avoids adding an empty path component and trailing slash if name is empty. This code should be simpler when this is rebased after #20932